js/tree.js

```diff
@@ -32,46 +34,57 @@
 	// TREE CONSTRUCTOR AND PROTOTYPE
 
 	var Tree = function Tree(element, options) {
-		this.$element = $(element);
+		var $element = $(element);
```

Comment:
I only see `$element` used once at https://github.com/ExactTarget/fuelux/pull/1984/files?#diff-e5768895bd9d9f525527077130cf278bR70

It could be replaced with 
```
this.$element.on('focus', function setFocusOnTab () {
	var $tree = $(this);
	focusIn($tree, $tree);
});
```

making this assignment unnecessary.